stack/app/requirements.txt

```diff
@@ -1,4 +1,4 @@
-timvt==0.8.0a2
-tifeatures==0.1.0a2
+timvt
+tifeatures
```

Comment:
knowing the person who maintains those 2 projects I would recommend to pin to specific version 😂

Reply:
I had some issues with the build when including the versions. Let me try it again and troubleshoot...

Reply:
I added versions to the requirements